linera-views/src/aggregator.rs

```diff
@@ -0,0 +1,366 @@
+// Copyright (c) Zefchain Labs, Inc.
```

Comment:
This module is specific to the Linera project (because it assumes that all the implementations of `KeyValueStoreClient` are known). Therefore, it belongs to linera-storage.

Reply:
Ok, moved.